src/backend/dx12/src/lib.rs

```diff
@@ -347,6 +352,7 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
     {
         // Reset idle fence and event
         // That's safe here due to exclusive access to the queue
+        // Nope.
```

Comment:
wut

Reply:
Uh. That was mark for me to notify someone that the statement above is incorrect.
It is not safe due to exclusive access to the queue because there is a clone in `Device`.
But later you told me that waiting and submitting must be externally synchronized. So it is indeed safe here 😄